contracts/AgglayerBridge.sol

```diff
@@ -66,7 +63,7 @@ contract AgglayerBridge is
     uint256 internal constant _GLOBAL_INDEX_MAINNET_FLAG = 2 ** 64;
 
     // Current bridge version
-    string public constant BRIDGE_VERSION = "v1.1.0";
+    string internal constant BRIDGE_VERSION = "v1.1.0";
```

Comment:
?¿

Reply:
v1.2.0 --> we are removing the initializer from previous version, joining the BytecodeStorer into the BirgdeLib and removing public functions